drivers/storage/digitalocean/storage/digitalocean_storage.go

```diff
@@ -0,0 +1,319 @@
+// +build !libstorage_storage_driver libstorage_storage_driver_digitalocean
+
+package storage
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strconv"
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/digitalocean/godo"
+
+	gofig "github.com/akutz/gofig/types"
+	"github.com/akutz/goof"
+	"github.com/codedellemc/libstorage/api/context"
+	"github.com/codedellemc/libstorage/api/registry"
+	"github.com/codedellemc/libstorage/api/types"
+	"github.com/codedellemc/libstorage/drivers/storage/digitalocean"
+	"github.com/codedellemc/libstorage/drivers/storage/digitalocean/utils"
```

Comment:
Hi @protochron,

Maybe import this as a non-generic utils alias?

```go
doUtils "github.com/codedellemc/libstorage/drivers/storage/digitalocean/utils"
```

That way it is clear what utils package is used when referencing it in the file.